.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: gh-actions/site
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ruby: [ '2.4', '2.5', '2.6', '2.7' ]
```

Comment:
Wouldn't be fine to run tests only on `2.6`? We don't need to test it on multiple versions IMO

Reply:
I don’t see why we shouldn’t. Maybe we can catch bugs this way as I don’t think everyone uses the same ruby version. I takes equally long and the computing power is free anyway.

Reply:
Fair enough. If it takes the same time, no problem.